include/gsl/string_span

```diff
@@ -108,18 +108,20 @@ namespace details
 // Will fail-fast if sentinel cannot be found before max elements are examined.
 //
 template <typename T, const T Sentinel>
-span<T, dynamic_extent> ensure_sentinel(T* seq, std::ptrdiff_t max = PTRDIFF_MAX)
+span<T, dynamic_extent> ensure_sentinel(T* seq,
+                                        std::size_t max = std::numeric_limits<std::size_t>::max())
 {
     Ensures(seq != nullptr);
 
-    GSL_SUPPRESS(f.23) // NO-FORMAT: attribute // TODO: false positive // TODO: suppress does not work
+    GSL_SUPPRESS(
+        f .23) // NO-FORMAT: attribute // TODO: false positive // TODO: suppress does not work
```

Comment:
// TODO: NO-FORMAT doesn't work? :trollface:

Reply:
dang... gotta go fix all of that.